connectors/es/sink.py

```diff
@@ -309,18 +313,27 @@ async def _run(self):
                         self._batch_bulk,
                         copy.copy(batch),
                         copy.copy(stats),
-                    )
+                    ),
+                    result_callback=functools.partial(self.batch_bulk_callback),
```

Comment:
I'd honestly just avoid callbacks and do it with try/except. I feel like callbacks add a lot of complexity while actually needed only for niche cases (such as events and all sorts of weak cohesion between unknown number of unknown actors)

Reply:
done